LICENSE

```diff
@@ -1,202 +1,180 @@
+
+  * "You" means an individual or legal entity exercising permissions granted by
+    this License.
+
+
+Use Rights; Scope of License
+----------------------------
+
+The Software is licensed on a per user basis. Here's what You may do
+with the Software, but subject to License Restrictions provisions
+below:
+
+  * Use the Software to perform academic research.
+
+  * Use the Software to demonstrate the Software for mere testing of the 
+    Software itself without production use.
+
+  * Test bug detectors or sanitizers that are released under an OSI-approved
+    License
+
+Here's what You may also do with the Software, but only with an Open
+Source Codebase and subject to the License Restrictions provisions
+below:
+
+  * Perform analysis on the Open Source Codebase but without automated
+    analysis / fuzzing, CI, or CD.
+
+  * Perform analysis on Open Source Codebases, including automated 
+    analysis / fuzzing, CI, or CD only through the OSS-Fuzz Infrastructure
+    operated by Google (https://github.com/google/oss-fuzz).
+
+License Restrictions
+--------------------
+
+These Terms do not authorize, and the Software may not be used for any
+purpose not expressly set forth above, including:
+
+  * To otherwise or in any other context, perform dynamic testing
+    for or during automated analysis, CI or CD, whether as part of
+    normal engineering processes or another context.
+
+  * To otherwise or in any other context, use the Software in
+    connection with any codebase that is not an Open Source Codebase
+    (e.g., code in a private repo in Github/Gitlab).
+
+_**Please note:** if Your use of the Software is under a paid customer
+license for Code Intelligence or Jazzer, the restrictions with respect to
+automated analysis, CI, and CD and use in connection with non-Open
+Source Codebases do not apply._
+
+At all times, except (and only to the extent) permitted by applicable
+law or applicable third-party license, You will not (and have no right
+to):
+
+  * work around any technical limitations in the Software that only
+    allow You to use it in certain ways;
+
+  * reverse engineer, decompile or disassemble the Software if provided in
+    binary form only;
+
+  * remove, minimize, block, or modify any notices of Code Intelligence 
+    or Contributors  in the Software;
+
+  * use the Software in any way that is against the law; or
+
+  * share, publish, distribute, rent, offer for download, or lend the Software, 
+    provide or make available the Software as a hosted solution (whether on a
+    standalone basis or combined, incorporated or integrated with other software
+    or services) for others to use, or transfer the Software to any third party.
+
+Code Intelligence reserves all rights not expressly granted in these Terms. 
+
+Open Source Software
+--------------------
+
+The Software may include components licensed under open source
+software licenses. Any such licenses are included in the "NOTICE"-file that
```

Comment:
The `NOTICE` file is missing from this PR.

Reply:
We added the line `This file also contains code licensed under Apache2 license.` to the headers of source files containing contributions from non-CI employees. This should be enough for now. Will add a NOTICE file later in a separate PR.